terraform/vars/delivered.tfvars

```diff
@@ -0,0 +1,5 @@
+vpc = "delivered"
+
+www_dns = "chef-web-docs-production"
+
+fastly_fqdn = "docs.chef.io"
```

Comment:
We have to be careful not to promote to production while this is in place. Maybe we comment it out?

Reply:
Good call, will do that now.